bioinformatics_toolkit/protein_tool.py

```diff
@@ -0,0 +1,218 @@
+from typing import Dict, List, Union
+import sys
```

Comment:
```suggestion
import sys
from typing import Dict, List, Union

```

Послед-ть импортов: сначала полные, потом частичные